react-components/src/components/BundleProductOptions/bundleProductOptions.js

```diff
@@ -0,0 +1,265 @@
+                                <span className={classes.customization__info}>
+                                    {t('product:customization-price', 'Your customization')}: {getTotalPrice()}
+                                </span>
+                            </h3>
+                        </section>
+                        <section className="productFullDetail__quantity productFullDetail__section">
+                            <h2 className="productFullDetail__quantityTitle option__title">
+                                <span>{t('cart:quantity', 'Quantity')}</span>
+                            </h2>
+                            <div className="quantity__root">
+                                <span className={'fieldIcons__root ' + classes.select_icons}>
+                                    <span className="fieldIcons__input">
+                                        <select
+                                            aria-label="product's quantity"
+                                            className="select__input field__input"
+                                            name="quantity">
+                                            <option value="1">1</option>
+                                            <option value="2">2</option>
+                                            <option value="3">3</option>
+                                            <option value="4">4</option>
+                                        </select>
+                                    </span>
+                                    <span className="fieldIcons__before"></span>
+                                    <span className="fieldIcons__after">
+                                        <span className="icon__root">
+                                            <svg
+                                                xmlns="http://www.w3.org/2000/svg"
+                                                width="18"
+                                                height="18"
+                                                viewBox="0 0 24 24"
+                                                fill="none"
+                                                stroke="currentColor"
+                                                strokeWidth="2"
+                                                strokeLinecap="round"
+                                                strokeLinejoin="round">
+                                                <polyline points="6 9 12 15 18 9"></polyline>
+                                            </svg>
+                                        </span>
+                                    </span>
+                                </span>
+                                <p className="message-root"></p>
+                            </div>
+                        </section>
+                        <section className="productFullDetail__cartActions productFullDetail__section">
+                            <button
```

Comment:
The event handler on the add to cart button via https://github.com/adobe/aem-core-cif-components/blob/master/ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/product/v1/product/clientlib/js/addToCart.js is only done once for when the whole document is loaded. Since this component starts with an initial `bundleState = null`, the button is not rendered and the event handler should not be set.

Reply:
the intention is to not use the default event handler for adding to cart since a bundle requires a different mutation.
I can modify the classes to avoid any confusion.

Reply:
but that would mean copying the CSS rules from `productFullDetail__cartActions` to another class and use that one instead in this button. not sure it's worth it